rotkehlchen/assets/converters.py

```diff
@@ -1031,3 +1033,18 @@ def asset_from_bitpanda(bitpanda_name: str) -> Asset:
 
     our_name = BITPANDA_TO_WORLD.get(bitpanda_name, bitpanda_name)
     return symbol_to_asset_or_token(our_name)
+
+
+def asset_from_cryptocom(crtptocom_name: str) -> Asset:
```

Comment:
```suggestion
def asset_from_cryptocom(cryptocom_name: str) -> Asset:
```

![giphy (2)](https://user-images.githubusercontent.com/1658405/148251994-cf32b82b-2c70-4d12-974f-c0b9383e2ef0.gif)